site/en/docs/extensions/reference/commands/index.md

```diff
@@ -110,3 +107,133 @@ Example:
   ...
 }
 ```
+
+## Examples
+
+The following examples show
+
+### Basic command
+
+Commands allow extensions to map logic to keyboard shortcuts that can be invoked by the user. At
+it's most basic, a command only requires a command declaration in the extension's manifest and a
+listener registration as shown below.
+
+```json
+// manifest.json
+{
+  "name": "Command demo - basic",
+  "version": "1.0",
+  "manifest_version": 3,
+  "background": {
+    "service_worker": "background.js"
+  },
+  "commands": {
+    "inject-script": {
+      "suggested_key": {
+        "default": "Ctrl+Shift+Y",
+        "mac": "Command+Shift+Y"
+      },
+      "description": "Inject a script on the page"
+    }
+  }
+}
+```
+
+```js
+// background.js
+chrome.commands.onCommand.addListener((command) => {
+  console.log(`Command "${command.name}" triggered`);
+});
+```
+
+### Action command
+
+As described in the [Usage][usage] section above, a command can also be mapped to the extension's
+action (Manifest V3), browser action (Manifest V2), or page action (Manifest V2). The below example
+programmatically injects a content script that shows an alert on the current page in response to the
+user either clicking the extension's action or triggering a keyboard shortcut.
+
+```json
+// manifest.json
+{
+  "name": "Commands demo - action invocation",
+  "version": "1.0",
+  "manifest_version": 3,
+  "background": {
+    "service_worker": "background.js"
+  },
+  "permissions": ["activeTab", "scripting"],
+  "action": {},
+  "commands": {
+    "_execute_action": {
+      "suggested_key": {
+        "default": "Ctrl+U",
+        "mac": "Command+U"
+      }
+    }
+  }
+}
+```
+
+```js
+// background.js
+chrome.action.onClicked.addListener((tab) => {
+  chrome.scripting.executeScript({
+    target: {tabId: tab.id},
+    func: contentScriptFunc,
+    args: ['action'],
+  });
+});
+
+function contentScriptFunc(name) {
+  alert(`"${name}" executed`);
+}
+
+// This callback WILL NOT be called for "_execute_action"
+chrome.commands.onCommand.addListener((command) => {
+  console.log(`Command "${command}" called`);
+});
+```
+
+### Verify commands registered
+
+If an extension attempts to register a shortcut that is already in use used by another extension,
+the second extension's shortcut will not register as expected. Developers can provide a more robust
```

Comment:
Consider "You can provide a more robust"

Reply:
Ack